.bazelrc

```diff
@@ -0,0 +1,2 @@
+build --cxxopt='-std=c++14'
+build --action_env SENTRY_DSN_PATH=/ --action_env SENTRY_DSN_HOST=localhost
```

Comment:
Can you set these two values to empty string? 

The sentry client treats that value as "disabled" as I understand it.

Reply:
Oh yes, let me validate. I was expecting an empty value to not be picked up properly during the process, so if it can be empty I'll set it that way.

Reply:
It works! I've pushed the update now.